onadata/apps/api/viewsets/data_viewset.py

```diff
@@ -653,76 +658,114 @@ def list(self, request, *args, **kwargs):
 
             return Response(serializer.data)
 
-        return custom_response_handler(request, xform, query, export_type)
+        return custom_response_handler(request, xform, query, export_type, sort=sort)
```

Comment:
I dont think its necessary to have a new `sort` param since you can get it from request